ui/appui-react/src/appui-react/framework/FrameworkContent.ts

```diff
@@ -196,6 +197,7 @@ export interface FrameworkContent {
   /**
    * Manage dialogs displaying managed content.
    * @beta
+   * @deprecated in 4.15.0. Use `modeless` or `modal` properties of {@link UiFramework.dialogs} instead.
    */
   readonly dialogs: FrameworkStackedDialog<ContentDialogInfo> & {
```

Comment:
@GerardasB do you think it is fine to deprecate this? I did it since we decided that content dialogs do not make sense and should just function as other modeless dialogs

Reply:
Yes, it should be fine.